spacemesh/v1/post_types.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package spacemesh.v1;
+option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
+
+import "spacemesh/v1/types.proto";
+
+// NodeRequest is a request sent from the node to the post service.
+message NodeRequest {
+  oneof type {
```

Comment:
`type` is a keyword in many languages (both Go and Rust) and it forces the code gen to workaround this (in rust the field name becomes `r#type` :( How about using "kind":
```suggestion
// NodeRequest is a request sent from the node to the post service.
message NodeRequest {
  oneof kind {
```